app/build.gradle

```diff
@@ -119,6 +131,7 @@ dependencies {
     androidTestImplementation libs.test.facebookSnapshotTestingCore
     implementation project(':Backpack')
     implementation project(':backpack-compose')
+    implementation project(":meta:annotations")
```

Comment:
Annotations have been extracted to separate module to be visible in tests runtime